extensions/iceberg/build.gradle

```diff
@@ -30,7 +30,6 @@ dependencies {
     implementation libs.iceberg.core
     implementation libs.iceberg.bundled.guava
 
-    implementation libs.hadoop.hdfs.client
```

Comment:
What did we need this for?

Reply:
@lbooker42 can you please help with this?

It seems to be added in the [very first iceberg commit](https://github.com/deephaven/deephaven-core/pull/5277/files#diff-61968a8c896c6b4d7e1c73c3678925f4dba0545e9389fbf3baceb5c20dd53912).
I tried checking out that commit and removing this dependency and the build completed successfully. So couldn't figure this out.

Reply:
I'm guilty of adding this but no longer sure of the reasoning behind it. I'm happy to see it removed.